web/src/pages/Settings/About/About.jsx

```diff
@@ -24,6 +33,10 @@ const About = ({ onClose }) => {
                 <br />
         Version: {info.version}
                 <br />
+        Main Unit Firmware Version: {info.fw.length ? Number(info.fw[0].version) : "No Preamp"} 
+                <br />
+        Expansion Unit Firmware Versions: {info.fw.length > 1 ? generateBoardVersionsString(info.fw): "No Expansion Units"}
```

Comment:
could you conditionally display this line? an example nearby is the `info.access_key` bit below.

Reply:
Would you like me to do so for both lines or just the expansion unit line?

Reply:
nvm lincoln explained what yall wanted me to do